src/NServiceBus.Core/Transports/TransportComponent.cs

```diff
@@ -67,13 +98,6 @@ public Task CreateQueuesIfNecessary(string username)
             return queueCreator.CreateQueueIfNecessary(QueueBindings, username);
         }
 
-        public QueueBindings QueueBindings { get; }
-
-        public IPushMessages BuildMessagePump()
```

Comment:
How where you able to remove this? (How do receive component get access to the message pump?)

Reply:
there is a GetMessagePump factory method used by the receive component a bit further up 🤷‍♂